File: app/src/main/java/com/babestudios/companieshouse/data/model/company/ConfirmationStatement.java
package com.babestudios.companieshouse.data.model.company;

import com.google.gson.annotations.SerializedName;

public class ConfirmationStatement {

	@SerializedName("last_made_up_to")
	public String lastMadeUpTo;

	@SerializedName("next_due")
	public String nextDue;

	@SerializedName("next_made_up_to")
	public String nextMadeUpTo;

	@SerializedName("overdue")
	public String overdue;

}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/company/ConfirmationStatement.kt
package com.babestudios.companyinfouk.data.model.company

import com.google.gson.annotations.SerializedName

class ConfirmationStatement {

	@SerializedName("last_made_up_to")
	var lastMadeUpTo: String? = null

	@SerializedName("next_due")
	var nextDue: String? = null

	@SerializedName("next_made_up_to")
	var nextMadeUpTo: String? = null

	@SerializedName("overdue")
	var overdue: String? = null

}